.env.example

```diff
@@ -130,8 +130,6 @@ NEXT_PUBLIC_TOKEN_ADDRESS=""
 # Do this if the schemas doesn't exist on the network you're using.
 WALLET_PRIVATE_KEY=""
 
-NEXT_PUBLIC_MACI_ADDRESS=
```

Comment:
Docs should be updated as well

Reply:
then does this still needed? `NEXT_PUBLIC_MACI_START_BLOCK` (I think we could remove it?)

Reply:
In this case you need to save maci contract deployed block to check the events staring from this block number.

Reply:
then maybe I open an issue and open another PR mainly for that?

Reply:
For the`NEXT_PUBLIC_MACI_START_BLOCK` it's fine.

Reply:
Open an issue for it: https://github.com/privacy-scaling-explorations/maci-rpgf/issues/115